pkgs/tools/security/vulnix/default.nix

```
@@ -1,17 +1,19 @@
 { lib
 , python3Packages
-, fetchPypi
+, fetchFromGitHub
 , nix
 , ronn
 }:
 
 python3Packages.buildPythonApplication rec {
   pname = "vulnix";
-  version = "1.10.1";
+  version = "unstable-2024-04-02";
```

```suggestion
  version = "1.10.1-unstable-2024-04-02";
```

https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-naming

---

Fixed in https://github.com/NixOS/nixpkgs/commit/b213b627a9c9db6a9610566ffb03eae08837b7da